ui/src/app/utils/resource-list.tsx

```diff
@@ -135,11 +135,6 @@ export const ResourcesList = fp.flow(withCdrVersions())((props: Props) => {
     });
   };
 
-  const getWorkspace = (r: WorkspaceResource) => {
```

Comment:
only used in one place, so I inlined it after adapting to the new `workspaces` prop